### ISSUE TITLE
Introduction to the Fediverse and ActivityPub

### ISSUE THREAD

**Issue by RangerMauve** [OttawaReactJS/talks#31]
This talk will go over ActivityPub and how it enables federated social media across different communities and server software.

- Intro to federated software
- Examples of software (Mastodon, Lemmy, Pixelfed)
- How ActivityPub works and formats
- What you can use this for

Ideally I'd like 30-45 mins for this as well as a projector with USB C

**Comment by btav** [comment]
I apologize for the delayed response! Would love to hear this talk! If possible could you also speak to the differences between ActivityPub and the AT Protocol used by Bluesky? 

Would you be available for the March or April meetup @RangerMauve?

**Comment by RangerMauve** [comment]
Hmm, yeah I could probably do that, how much time can I get to cover everything? Would 40 mins be possible?

**Comment by macdonst** [comment]
Hey @RangerMauve this looks like a great talk and we'd love to have you do it at an upcoming meetup we are just not sure what date it will be. January is a social and February's talks are set. We could end up turning to you as early as March or a little later in the Spring if you are game?

**Comment by RangerMauve** [comment]
YEah sure, march or spring works for me. Just give me a couple weeks warning if possible :)

**Comment by macdonst** [comment]
Will do!

**Comment by RangerMauve** [comment]
I feel like we had scheduled this at some point but I'm not seeing anything on my calendar. Is this still on your radar?

**Comment by macdonst** [comment]
It's still on our radar. We are having trouble tracking down venues that are able to host which is why we haven't reached out. Stay tuned.

**Comment by btav** [comment]
Hey @RangerMauve apologize for the delay! Would you be available to speak at the June meetup on the 19th?

**Comment by RangerMauve** [comment]
@btav Yup, works for me! How long can I aim for?

**Comment by btav** [comment]
Does ~30-35mins work for you? (including a bit of time for questions we use slido to get questions from the audience)

**Comment by RangerMauve** [comment]
30 mins should be good! I'm gonna cover quite a bit so watch out :P

**Comment by btav** [comment]
Awesome sounds good I will update the event on Meetup! Looking forward to it!

**Comment by btav** [comment]
Will the talk be titled "Introduction to the Fediverse and ActivityPub"?

**Comment by RangerMauve** [comment]
Lets rename it "Overview of Federated social media" since I also want to get into bluesky and nostr

**Comment by RangerMauve** [comment]
Oh oh, actually: "Decentralized Social Media: What is it, how does it work?"

**Comment by btav** [comment]
Great talk @RangerMauve!! We have the recording on the Google meetup calendar link provided by fullscript. You might just need to request permissions to view/download the video.
